include/tvm/runtime/vm.h

```diff
@@ -506,6 +574,9 @@ class VirtualMachine : public runtime::ModuleNode {
   /*! \brief The special return register. */
   ObjectRef return_register;
 
+  /*! \brief The executable the VM will operate on. */
+  const Executable* exec;
```

Comment:
should we move these fields (packed_funcs, frames, func_index, return_register, exec) to private?

Reply:
yeah, this is similar to the other fields in `VirtualMachine`, we want to use the fields in VMCompiler as well. We may want to add setter then.

Reply:
oops. I thought you were talking about `Executable`.  Will do.